src/components/profile-link.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class ProfileLink extends Component {
+	resolveHref() {
+		let { custom, href } = this.props;
+
+		if ( custom && href ) {
+			href = `${ custom }:${ href }`;
+		}
+
+		return href;
+	}
+
+	content() {
+		return this.props.children || this.props.href;
+	}
+
+	render() {
+		return (
+			<a href={ this.resolveHref() }>{ this.content() }</a>
+		);
+	}
+
+	static get propTypes() {
+		return {
+			href: React.PropTypes.string
+		};
```

Comment:
This cannot be required as the value might be `null`, e.g.: if the employee has not a registered phone.